File: hard/6_increasing_subsequence/increasing_subsequence.test.js
const  MaxSumIncreasingSubsequence = require('./increasing_subsequence');
const assert = require('assert')

test("Test Case #1", () => {
  const [outputSum, outputSequence] = MaxSumIncreasingSubsequence([10, 70, 20, 30, 50, 11, 30]);
  expect(outputSum).toEqual(110);
  expect(outputSequence).toEqual([10, 20, 30, 50]);
});

File: hard/6_increasing_subsequence/increasing_subsequence_test.py
import unittest
from increasing_subsequence import maxSumIncreasingSubsequence

class TestProgram(unittest.TestCase):
    def test_case_1(self):
        outputSum, outputSequence = maxSumIncreasingSubsequence([10, 70, 20, 30, 50, 11, 30])
        self.assertEqual(outputSum, 110)
        self.assertEqual(outputSequence, [10, 20, 30, 50])
